spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -89,12 +89,12 @@ headers::
   The set of custom headers to transfer over AWS Kinesis
 +
 Default: "correlationId", "sequenceSize", "sequenceNumber", "contentType", "originalContentType".
-describeStreamBackoff::
```

Comment:
I see we still use a `DescribeStream` API in the provisioner.
So, why have you removed its properties from here?